src/java/fr/paris/lutece/plugins/suggest/business/attribute/SuggestAttributeHome.java

```diff
@@ -52,6 +52,14 @@
     private static Plugin _plugin = PluginService.getPlugin( SuggestPlugin.PLUGIN_NAME );
     private static ISuggestAttributeDAO _dao = SpringContextService.getBean( BEAN_SUGGEST_ATTRIBUTE_DAO );
 
+    /**
+     * Private constructor - this class needs not to be instantiated
```

Comment:
Wouldn't "does not need to..." be better ?

Reply:
Yes, certainly.